hawkular-component/src/main/java/org/hawkular/metrics/security/InfluxAuthHttpHandler.java

```diff
@@ -68,9 +73,7 @@ public void handleRequest(HttpServerExchange httpServerExchange) throws Exceptio
         HeaderMap requestHeaders = httpServerExchange.getRequestHeaders();
 
         String databaseName = matcher.group(1);
-        try {
```

Comment:
OK, I'll simply revert the second commit then.

Le 05/02/2016 11:21, Juraci Paixão Kröhling a écrit :
> In
> hawkular-component/src/main/java/org/hawkular/metrics/security/InfluxAuthHttpHandler.java
> <https://github.com/hawkular/hawkular-metrics/pull/444#discussion_r51998297>:
>
>> @@ -68,9 +73,7 @@ public void handleRequest(HttpServerExchange httpServerExchange) throws Exceptio
>>          HeaderMap requestHeaders = httpServerExchange.getRequestHeaders();
>>
>>          String databaseName = matcher.group(1);
>> -        try {
>
> The regex above is not meant to replace this try/catch. It might be the
> case that it matches the regex but still is not spec-compliant. In such
> case, Accounts will throw a RuntimeException, so, it's a matter of taste
> now :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hawkular/hawkular-metrics/pull/444/files#r51998297>.
>